games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="3D kart racing game"
+DESCRIPTION="SuperTuxKart is a free 3D kart racing game. It aims towards fun for 
+players of all skill levels, with item boxes giving random items, nitro, drifting and 
+more. Realism is not the focus.
+
+STK offers an online multiplayer mode, a local multiplayer mode, and a single player \
+mode against AIs with both custom races and a story mode to complete to unlock new \
+karts and tracks. It features Grand Prix where the goal is to get the most points over \
+several races.
+
+The game also features additional game modes besides normal races : time trials, \
+follow-the-leader, soccer, capture-the-flag and two types of battle mode."
+HOMEPAGE="https://supertuxkart.net/
+	https://github.com/supertuxkart"
+COPYRIGHT="2020 SuperTuxKart Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/$portName/stk-code/releases/download/$portVersion/$portName-$portVersion-src.tar.xz"
+CHECKSUM_SHA256="052edf0afdbeb99583fe8676fb0ab80ecb6103fb88b7540f858d1b5fa1297d37"
+SOURCE_FILENAME="supertuxkart-$portVersion-src.tar.xz"
+SOURCE_DIR="SuperTuxKart-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	supertuxkart$secondaryArchSuffix = $portVersion
+	app:SuperTuxKart$secondaryArchSuffix
+	"
+
+REQUIRES="
```

Comment:
You should check the gentoo recipe for the dependencies and cmake switches (but no the X-server related stuff!) : https://gitweb.gentoo.org/repo/gentoo.git/tree/games-action/supertuxkart/supertuxkart-1.2.ebuild